### ISSUE TITLE
EXC_BAD_ACCESS in IBGDataStore

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/Instabug-iOS#175]
```
EXC_BAD_ACCESS Attempted to dereference garbage pointer 0x91f1fbec8. 
    /usr/lib/libobjc.A.dylib objc_retain
    Instabug/Logging/IBGDataStore.m:569:9 -[IBGDataStore addEventLogWithName:params:]
    Instabug/Instabug.m:309:9 __40+[Instabug logUserEventWithName:params:]_block_invoke
    Frameworks/Foundation.framework/Foundation ___NSBLOCKOPERATION_IS_CALLING_OUT_TO_A_BLOCK__
    Frameworks/Foundation.framework/Foundation -[NSBlockOperation main]
    Frameworks/Foundation.framework/Foundation -[__NSOperationInternal _start:]
    Frameworks/Foundation.framework/Foundation ___NSOQSchedule_f
    /usr/lib/system/libdispatch.dylib __dispatch_client_callout
    /usr/lib/system/libdispatch.dylib __dispatch_queue_serial_drain
    /usr/lib/system/libdispatch.dylib __dispatch_queue_invoke
    /usr/lib/system/libdispatch.dylib __dispatch_root_queue_drain
    /usr/lib/system/libdispatch.dylib __dispatch_worker_thread3
    /usr/lib/system/libsystem_pthread.dylib __pthread_wqthread
```

Instabug sdk 7.3.2

**Comment by Kmohamed** [comment]
@SudoPlz We are aware of this crash and it will be fixed in the next release.

**Comment by Kmohamed** [comment]
@SudoPlz This crash is fixed on the latest version 7.3.6
